strawberryfields/ops.py

```diff
@@ -335,8 +335,8 @@ class Operation:
     :class:`~strawberryfields.parameters.Parameter` class for more details.
 
     Args:
-        par (Sequence[...]): parameters. len(par) >= 1. Alternatively,
-            set to [] or None to allow initialisation with no parameters.
+        par (Sequence[Any]): Operation parameters. An empty sequence if no parameters
+            are required.
```

Comment:
I haven't seen any piece of code using it, and the tests run ok. Since an empty list already means no parameters it's not a good idea to use None to mean the same thing.

Reply:
thumb